docs/eigenda/operator-guides/troubleshooting.md

```diff
@@ -76,4 +76,12 @@ again.
 
 #### What does the error "EIP1271 .. signature not from signer" mean?
 
-This indicates you have not imported your BLS key correctly. Please reconfirm the keys you imported to ensure there were no typos or mistakes.
\ No newline at end of file
+This indicates you have not imported your BLS key correctly. Please reconfirm the keys you imported to ensure there were no typos or mistakes.
+
+#### Error message "failed to update operator's socket .. execution reverted"
```

Comment:
execution reverted imo does not happen if rpc is down? what you mean rpc is not functioning correct?

Reply:
I think `execution reverted` can also occur when the operator config is broken ie trying to call an old goerli contracts on holesky/mainnet or using wrong churner (testnet, mainnet).

Reply:
Hi @shrimalmadhur are we ok to resolve this comment? Or are there any changes you'd like for me to make?

Reply:
just saying it could be both problems as above would suffice

Reply:
Sounds good - I've added the following "This indicates the RPC endpoint may not be functioning correctly or the operator config is misconfigured (eg pointing to the wrong chain_id value)."